plugins/readtimer.koplugin/main.lua

```diff
@@ -324,19 +399,22 @@ function ReadTimer:onShowAlarm(touchmenu_instance)
     local time_widget = DateTimeWidget:new{
         hour = curr_hour,
         min = curr_min,
-        ok_text = _("Set alarm"),
-        title_text =  _("New alarm"),
+        ok_text = _("Start timer"),
```

Comment:
Is this change congruent with that other one?

<img width="504" height="130" alt="image" src="https://github.com/user-attachments/assets/8670b4cc-0791-4079-9abf-589dc9d87584" />

Reply:
It's a recent change after heavily testing the new features.
What do we want and what does this SpinWidget do? We want to start the timer, not just set the alarm/interval.

<img width="400" height="540" alt="5" src="https://github.com/user-attachments/assets/3ac5561b-0a7d-4b50-b421-cf3ce5fe5d19" />

Reply:
I'd say an alarm is not a timer (even if it acts as an x-minute timer under the hood) and the checkbox is just a side feature.

The interval is a timer though and that's probably a better word to use.

Reply:
But also "Stop timer" for both.

Do you recommend to keep "Set alarm" and "Set interval" for spinwidget OK buttons?

Reply:
I remember the @offset-torque 's post about the timer and its wordings that they were not perfect.

Reply:
> In the footer, we actually show the countdown timer in both cases.

I just checked and status bar shows either the "alarm countdown" or the "timer countdown".

Think as a new user:

> I want to set an alarm. I clicked the "Set alarm" button. I didn't click the "Set timer" button. Why am I seeing "Show timer in status bar"?!? I should see a countdown to my alarm that I am currently setting. 

Which is what that checkboxes do actually. Just named incorrectly now.

"Set timer" dialog's checkboxes are (nearly) correct because they show the countdown to the "timer" which is set in that dialog.

Clearer and more accurate wording for them is:
- "Show countdown to alarm in status bar" 
- "Show countdown to timer in status bar"

But it is more crowded.